sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubConsumer.cs

```diff
@@ -94,19 +98,21 @@ public class EventHubConsumer : IAsyncDisposable
         internal EventHubConsumer(TransportEventHubConsumer transportConsumer,
```

Comment:
Earlier in the thread, python and js have consumer group before partition id. @ramya-rao-a mentioned that .NET's first client library had the same order of parameters.

Reply:
Bah.   It makes more sense to me in this order, but so be it.

Reply:
Ahaha me too. But.. yup. 😢